queries/weather.py

```diff
@@ -71,11 +71,20 @@
     | "hvernig" "er" "veðurspáin" QWeatherLocation? QWeatherNextDays?
     | "hvernig" "er" "spáin" QWeatherLocation? QWeatherNextDays?
     | "hver" "er" "veðurspá" QWeatherLocation? QWeatherNextDays?
+    | "hver" "er" "veðurspáin" QWeatherLocation? QWeatherNextDays?
     | "hvernig" "er" "veðrið" QWeatherLocation? QWeatherNextDays
     | "hvernig" "verður" "veðrið" QWeatherLocation? QWeatherNextDays
     | "hvernig" "eru" "veðurhorfur" QWeatherLocation? QWeatherNextDays?
+    | "hvernig" "eru" "veður" "horfur" QWeatherLocation? QWeatherNextDays?
```

Comment:
This could be made more efficient by having a separate nonterminal for the question form ("hvernig er/eru") and a separate nonterminal for the noun phrase ("veðrið", "veðurhorfur", "veðurhorfurnar", "veður" "horfurnar" etc.)